include/circt/Dialect/FIRRTL/Passes.td

```diff
@@ -47,7 +47,10 @@ def LowerFIRRTLTypes : Pass<"firrtl-lower-types", "firrtl::CircuitOp"> {
     Option<"flattenAggregateMemData", "flatten-mem", "bool", "false",
            "Concat all elements of the aggregate data into a single element.">,
     Option<"preserveAggregate", "preserve-aggregate", "bool", "false",
-           "(Experimental) Preserve aggregate vectors/bundles if possible.">
+           "(Experimental) Preserve aggregate vectors/bundles if possible.">,
```

Comment:
Don't mark with experimental.  If it's in the tree, it should work and we should support it.

Reply:
Thanks. Addressed in https://github.com/llvm/circt/commit/f40cf9418a798e02d05447403b1116b65566d5a7